src/Core/functions.php

```diff
@@ -74,7 +74,7 @@ function h($text, bool $double = true, ?string $charset = null)
      * @param string $name The name you want to plugin split.
      * @param bool $dotAppend Set to true if you want the plugin to have a '.' appended to it.
      * @param string|null $plugin Optional default plugin to use if no plugin is found. Defaults to null.
-     * @return array Array with 2 indexes. 0 => plugin name, 1 => class name.
+     * @return string[] Array with 2 indexes. 0 => plugin name, 1 => class name.
```

Comment:
This incorrect since plugin name (value at 0 index) can be `null`. Psalm also complains about the same.

Reply:
Ah ok